### PR TITLE
Improvements to the web server

### PR DESCRIPTION
**Pull request by HeikoTheissen**
This PR serves to collect improvements to the web server:
* Don't use `iterator.js` during start-up so that new documents can be added while the server runs.
* Include SVG during build, not as images at runtime

### CODE REVIEW

**Review by ralfhandl — APPROVED** [review]

File: lib/number.js [hunk 1]
```diff
@@ -137,16 +137,25 @@ class Number {
               var m1 = line.match(/^\$\$\$(.*?isec)\$\$\$$/);
               if (m1) this.tableofcontents(this.toc[m1[1]]?.sub || [], out, "");
               else {
+                m1 = line.match(/^\$\$\$include\s+(.*?)\$\$\$$/);
```
Comment (ralfhandl) [inline lib/number.js:140]: No trailing whitespace allowed after the closing `$$$`? That could be hard to spot.

Comment (ralfhandl) [reply lib/number.js:140]: If the images are embedded in the Markdown and HTML files, then we don't need to copy the `images` subfolder to `/docs`.

Comment (HeikoTheissen) [reply lib/number.js]: Perhaps we also need *real* images later.

File: odata-temporal-ext/svg/api-1.drawio.svg
Comment (ralfhandl) [inline odata-temporal-ext/svg/api-1.drawio.svg:1]: I'd prefer `diagrams` as folder name to be consistent with Data Aggregation

File: lib/README.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ The [`number.js`](number.js) module generates a single Markdown document by prep
 - Generate section and example numbers
 - Generate a table of contents
 - Resolve references
+- Include files like `$$$include images/drawing.svg$$$`
```
Comment (ralfhandl) [inline lib/README.md:25]:
```suggestion
- Include files like `$$$include diagrams/drawing.svg$$$`
```

Comment (ralfhandl) [inline lib/README.md:25]:
```suggestion
- Copy `images` subfolder to output folder
- Include files like `$$$include diagrams/drawing.svg$$$`
```

**Review by ralfhandl — APPROVED** [review]

**Pull request merged**